python/sglang/srt/layers/attention/hybrid_linear_attn_backend.py

```diff
@@ -552,14 +567,27 @@ def forward_decode(
         query_start_loc = self.forward_metadata.query_start_loc
         cache_indices = self.forward_metadata.mamba_cache_indices
 
-        mixed_qkv = causal_conv1d_update(
-            mixed_qkv,
-            conv_states,
-            conv_weights,
-            bias,
-            activation,
-            conv_state_indices=cache_indices,
-        )
+        if _use_cpu:
+            mixed_qkv = causal_conv1d_update(
+                mixed_qkv,
+                conv_states,
+                conv_weights,
+                bias,
+                activation == "silu",
+                None,
+                cache_indices,
+                self.pad_slot_id,
+                True,
+            )
+        else:
+            mixed_qkv = causal_conv1d_update(
+                mixed_qkv,
+                conv_states,
+                conv_weights,
+                bias,
+                activation,
+                conv_state_indices=cache_indices,
+            )
```

Comment:
```suggestion

            mixed_qkv = causal_conv1d_update(
                mixed_qkv,
                conv_states,
                conv_weights,
                bias,
                activation,
                conv_state_indices=cache_indices,
```
is it possible to wrap this function in `sgl-kernel/python/sgl_kernel/mamba.py`, and skip the `if _use_cpu()` here